Cargo.toml

```diff
@@ -98,6 +99,10 @@ tokio-util = "0.7.10"
 url = "2.4.1"
 urlpattern = "0.2.0"
 wasm-bindgen = "0.2.92"
+bollard = "0.16.1"
```

Comment:
nit: dependencies are supposed to be sorted alphabetically

Reply:
done